Squeezer/src/main/java/uk/org/ngo/squeezer/NowPlayingFragment.java

```diff
@@ -578,24 +580,33 @@ private void updatePlayerDropDown(@NonNull List<Player> connectedPlayers, @Nulla
             actionBar.setDisplayShowTitleEnabled(false);
             actionBar.setDisplayShowCustomEnabled(true);
             actionBar.setCustomView(R.layout.action_bar_custom_view);
-            AutoCompleteTextView spinner = actionBar.getCustomView().findViewById(R.id.player);
+            AppCompatSpinner spinner = actionBar.getCustomView().findViewById(R.id.player);

```

Comment:
There is no Spinner in Material Design.
Spinner doesn’t follow the Material guidelines, and in this implementation the current selection not visible.
This comment also applies to Alarm Playlists.

Reply:
Right. [Others have also noticed that](https://rmirabelle.medium.com/there-is-no-material-design-spinner-for-android-3261b7c77da8) - apparently the 'official' way is to use a styled `AutoCompleteTextView`, just as you're doing, but doing so fails the [duck test](https://en.wikipedia.org/wiki/Duck_test) in my eye: for all intents and purposes in this app, the player selection dropdown as used in this app *is* a spinner, just looking more ugly due to the selection background in light theme ... and the point of my change is just to get rid of the selection highlight + frame in light theme (neither of which is visible in dark theme).

Reply:
This is what Android gives us, so I'm afraid we have to live with exposed drop down menus. The use of Spinner exposes you to a myriad of issues. Besides those mentioned above, Spinners have the bad habit of initially firing onItemSelected even if the user did not select anything. I see you attempt to handle that by comparing the active player from the service with the one from the selection. While this seems ok, I've previously had issues when this fires when the connection is reset due to communication issues. By not using Spinners at all, we do not expose ourselves to these issues.
Regarding whether it _is_ a `Spinner` to me that word indicates something that spins, so not a drop-down menu. Of course, if you interpret `Spinner`to mean what it meant when you first encountered it as an Android widget, then you are correct about the _dock test_.
In my opinion, Spinners doesn't really look good in a Material Design context, see below. I agree though that the exposed dropdown menu for player select doesn't look good in the light theme either. If you can't be bothered with it, I suggest you remove the Spinners from this PR (player drop down and alarm playlists), and then I will see if I can do anything about the highlight+frame. 
![image](https://github.com/kaaholst/android-squeezer/assets/428304/b40bec7e-7adc-49b7-bc7f-ac5c92d52d46)